src/mpi/coll/op/oputil.c

```diff
@@ -5,6 +5,28 @@
 
 #include "mpiimpl.h"
 
+/* The order of entries in this table must match the definitions in
+   mpi.h.in */
+MPI_User_function *MPIR_Op_table[] = {
+    NULL, MPIR_MAXF,
+    MPIR_MINF, MPIR_SUM,
+    MPIR_PROD, MPIR_LAND,
+    MPIR_BAND, MPIR_LOR, MPIR_BOR,
+    MPIR_LXOR, MPIR_BXOR,
+    MPIR_MINLOC, MPIR_MAXLOC,
+    MPIR_REPLACE, MPIR_NO_OP
+};
+
+MPIR_Op_check_dtype_fn *MPIR_Op_check_dtype_table[] = {
+    NULL, MPIR_MAXF_check_dtype,
+    MPIR_MINF_check_dtype, MPIR_SUM_check_dtype,
+    MPIR_PROD_check_dtype, MPIR_LAND_check_dtype,
+    MPIR_BAND_check_dtype, MPIR_LOR_check_dtype, MPIR_BOR_check_dtype,
+    MPIR_LXOR_check_dtype, MPIR_BXOR_check_dtype,
+    MPIR_MINLOC_check_dtype, MPIR_MAXLOC_check_dtype,
+    MPIR_REPLACE_check_dtype, MPIR_NO_OP_check_dtype
+};
+
```

Comment:
I think the move makes sense. Thanks. Could you additionally re-format the two initializers so each entry is on their own line?

Reply:
Do you mean to have each operator occupying one line?

Reply:
Yes. It is more readable and editable that way.